### PR TITLE
RDM-10737 Update FTA test data for use with befta-fw containing "Switchable Default for Collection Assertion"

### PR DESCRIPTION
**Pull request by sullivangareth**
JIRA link (if applicable)
https://tools.hmcts.net/jira/browse/RDM-10737

Change description
Update test data for datastore to use latest befta-fw pre-release

Does this PR introduce a breaking change? (check one with "x")

[ ] Yes
[X] No

### CODE REVIEW

**Review by mattnayler — CHANGES_REQUESTED** [review]
LGTM 👍 .  However please confirm or reset relevant Jenkins and chart changes prior to merger.

File: Jenkinsfile_nightly [hunk 1]
```diff
@@ -79,8 +79,8 @@ env.OAUTH2_REDIRECT_URI = "https://www-ccd.aat.platform.hmcts.net/oauth2redirect
 env.DM_STORE_BASE_URL = "http://dm-store-aat.service.core-compute-aat.internal"
 env.RD_PROFESSIONAL_API_BASE_URL = "http://rd-professional-api-aat.service.core-compute-aat.internal"
 env.BEFTA_RESPONSE_HEADER_CHECK_POLICY="JUST_WARN" // Temporary workaround for platform changes: turn BEFTA header checks to warning mode
-env.ELASTIC_SEARCH_FTA_ENABLED = "false"
-
+env.ELASTIC_SEARCH_FTA_ENABLED = "true"
```
[the remark this answers is not in the document]
Comment (sullivangareth) [reply Jenkinsfile_nightly:82]: No, I will change this back

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -91,7 +91,8 @@ env.OAUTH2_REDIRECT_URI = "https://www-ccd.aat.platform.hmcts.net/oauth2redirect
 env.DM_STORE_BASE_URL = "http://dm-store-aat.service.core-compute-aat.internal"
 env.RD_PROFESSIONAL_API_BASE_URL = "http://rd-professional-api-aat.service.core-compute-aat.internal"
 env.BEFTA_RESPONSE_HEADER_CHECK_POLICY="JUST_WARN"
-env.ELASTIC_SEARCH_FTA_ENABLED = "false"
+env.ELASTIC_SEARCH_FTA_ENABLED = "true"
```
[the remark this answers is not in the document]
Comment (sullivangareth) [reply Jenkinsfile_CNP:94]: Yes, I will revert this back to its original value

File: Jenkinsfile_CNP [hunk 3]
```diff
@@ -17,8 +17,8 @@ def component = "data-store-api"
 
 // Variables to switch pipeline logic and wiring per type of build
 
-def definitionStoreDevelopPr = "PR-575" // This doesn't change frequently, but when it does, only change this value.
-def dataStoreApiDevelopPr    = "PR-1260" // This doesn't change frequently, but when it does, only change this value.
+def definitionStoreDevelopPr = "PR-948" // This doesn't change frequently, but when it does, only change this value.
+def dataStoreApiDevelopPr    = "PR-1380" // This doesn't change frequently, but when it does, only change this value.
```
[the remark this answers is not in the document]
Comment (sullivangareth) [reply Jenkinsfile_CNP:21]: Yes

File: charts/ccd-data-store-api/values.preview.template.yaml [hunk 4]
```diff
@@ -16,7 +16,7 @@ java:
     DATA_STORE_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified"
     DATA_STORE_DB_MAX_POOL_SIZE: 10
-    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal/
+    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-948.service.core-compute-preview.internal/
```
[the remark this answers is not in the document]
Comment (sullivangareth) [reply charts/ccd-data-store-api/values.preview.template.yaml:19]: Yes they will

**Review by mattnayler — APPROVED** [review]
Approved **subject to [my previous comments](https://github.com/hmcts/ccd-data-store-api/pull/1380#pullrequestreview-596547297) being addressed as required during checkpoint**

**Review by MSancaktutar — APPROVED** [review]
LGTM, but please push a commit just before merging to restore the wiring and ES-enablement changes.

**Pull request merged**